geiger/src/graph/NodeWithPort.js

```diff
@@ -242,6 +242,11 @@ export default class NodeWithPort extends Node {
     return ports;
   }
 
+  /**
+   * Validate ports created and then store.
+   * @param {!Array<Port>} ports
```

Comment:
Where is `Port` is defined?

Reply:
`Port` is defined at the beginning of the file, right above the class comments.